spring-cloud-gcp-data-datastore/src/main/java/com/google/cloud/spring/data/datastore/core/mapping/DatastorePersistentProperty.java

```diff
@@ -69,4 +69,13 @@ public interface DatastorePersistentProperty
    * @return {@code true} if the property is lazily-fetched. {@code false} otherwise.
    */
   boolean isLazyLoaded();
+
+  /**
+   * Return whether to skip null value, i.e., skip insertion if value is null.
+   *
+   * @return {@code true} if the null value is skipped. {@code false} otherwise.
+   */
+  default boolean isSkipNullValue() {
+    return true;
```

Comment:
I thought default behavior is false? https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/30af5dee266bc8cf3f9993640461533748b4a661/docs/src/main/asciidoc/datastore.adoc?plain=1#L82

Am I missing something obvious?

Reply:
Good catch, this should be default to `false`.